.travis.yml

```diff
@@ -1,13 +1,18 @@
 language: ruby
+cache: bundler
 before_script: bundle install
-script: bundle exec rake
+script: bundle exec rake test
 rvm:
   - 2.0.0
-  - 2.1.8
   - 2.2.4
   - 2.3.1
-  - ruby-head
```

Comment:
I gave the reason of this being a moving target in the commit message but there is also the issue that for some reason I cannot seem to install this.